uefi/src/proto/network/mod.rs

```diff
@@ -34,3 +34,40 @@ impl IpAddress {
         Self(ip_addr)
     }
 }
+
```

Comment:
@nicholasbishop  Is there a reason why this type is not yet in `uefi-raw`?

Reply:
Filed an issue about our IP address types: https://github.com/rust-osdev/uefi-rs/issues/1575